packages/teams-js/src/public/interfaces.ts

```diff
@@ -1262,3 +1262,41 @@ export interface ClipboardParams {
   /** Blob content in Base64 string format */
   content: string;
 }
+
+/**
+ * @hidden
+ * List of enter focus action items
+ *
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export enum EnterFocusActionItem {
```

Comment:
Since this is in the interface naming is of importance. Action Item implies a queued lambda or an unprocessed queued action. Is there a different name that would intuitively suggest enumerated choices for Focus stops?

Reply:
How about `EnterFocusType` and `ReturnFocusType` .?